tests/conftest.py

```diff
@@ -642,3 +642,16 @@ def group_data(orders, status):
 def collection(db):
     collection = Collection.objects.create(name='Collection', slug='collection')
     return collection
+
+
+@pytest.fixture
+def product(product_type, default_category):
```

Comment:
We already have the `product_in_stock` fixture that creates a product. Doesn't it work for your case?

Reply:
It works, missed that one.